data/json/items/toy.json

```diff
@@ -0,0 +1,131 @@
+[
+  {
+    "type": "GENERIC",
+    "id": "toy_plush",
+    "name": { "str": "plush toy" },
+    "description": "A manufactured plush toy.  This is the default variant, if you see it something went wrong.",
+    "category": "other",
+    "material": [ "cotton" ],
+    "weight": "900 g",
+    "volume": "3 L",
+    "price": 2000,
+    "price_postapoc": 100,
+    "symbol": "i",
+    "color": "cyan",
+    "flags": [ "SLEEP_AID" ],
+    "variant_type": "generic",
+    "variants": [
+      {
+        "id": "toy_plush_teddy",
+        "name": { "str": "teddy bear" },
+        "description": "A mass-produced plush teddy bear.  It is wearing a little red t-shirt but no pants.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_teddy_zombie",
+        "name": { "str": "zombie teddy bear" },
+        "description": "A dark green teddy bear with stylized stitches all over and X's for eyes.  The perfect size for a small child to hug the unlife out of.  Looks like the zombie apocalypse hit the toy store shelves first.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_ferret",
+        "name": { "str": "ferret plushie" },
+        "description": "A large plush ferret with cute beady eyes.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_trex",
+        "name": { "str": "T-rex plushie" },
+        "description": "A large plush t-rex.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_penguin",
+        "name": { "str": "penguin plushie" },
+        "description": "A velvety-soft plush penguin.  It makes a funny noise when you hug it.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_rabbit",
+        "name": { "str": "rabbit plushie" },
+        "description": "A large plush rabbit.",
+        "weight": 1
+      },
+      {
+        "id": "toy_plush_remora",
+        "name": { "str": "remora plushie" },
+        "description": "A large plush remora, a fish commonly seen alongside sharks.  Wait, wasn't there a shark plushie somewhere there…",
+        "weight": 1
+      }
+    ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "shark_plushie",
```

Comment:
Please don't move data around arbitrarily. It hides changes you make to it and destroys history.

Reply:
In the past, i.e. #61573, I was told to move items to a new file when the specific item fits a new file better and doesn't necessarily fit in the file it is in now. I decided not to move all the toy items, just the shark plushie because having all plushies in one spot would be beneficial to anyone trying to contribute to that in the long term.

Reply:
Okay, I agree with the move in #65173, and I think it's fine here. In general I'd be reticent about it for those reasons, however.

Reply:
In most of my PRs, I try to avoid this like fire, though I think that having two separate plushie items in two separate files could possibly end up in a duplicate item if not handled like this. I will, like I said before though, not move the other toy items because they should hopefully be fine where they are and not cause problems in future contributions.